src/components/BackButton/BackButton.tsx

```diff
@@ -0,0 +1,27 @@
+import { StyledBackButton } from './StyledBackButton';
+import { ButtonHTMLAttributes } from 'react';
+import { useNavigate } from 'react-router';
+
+export interface BackButtonProps
```

Comment:
This doesn't need to be exported unless you plan to use it elsewhere.

```suggestion
interface BackButtonProps
```